src/material/tabs/testing/tab-link-harness.ts

```diff
@@ -33,6 +33,11 @@ export class MatTabLinkHarness extends ComponentHarness {
     return (await this.host()).text();
   }
 
+  /** Gets the aria-label of the tab. */
+  async getAriaLabel(): Promise<string | null> {
```

Comment:
I'm not sure that we need a dedicated method for this since you can get it with existing public methods (`host().getAttribute('aria-label')`). We have a few other harnesses with `getAria*` methods, but in those cases the elements with the ARIA attributes are a bit harder to reach.